source/newcommitter.md

```diff
@@ -140,178 +150,54 @@ with no holds barred. Keep the discussion concise. The praise part can
 be done later in public. Keep in mind, however, that if the member becomes
 a PMC member later, they will have access to this discussion.
 
-Let the Vote thread run for one week.
+### Vote
+
+If the proposed candidate seems to be received positively by a majority
+of those responding, it's time to start a vote.
```

Comment:
Instead of referring to the templates as in the next paragraph, I think it makes it easier to read if you refer directly to the templates from the action. This makes it more concise.

```suggestion
of those responding, it's time to [start a vote](templates/committer-vote.txt).
```